ui/plugins/controller.js

```diff
@@ -339,7 +339,20 @@ treeherder.controller('PluginCtrl', [
                    ($scope.job.state === "pending" || $scope.job.state === "running");
         };
 
-        $scope.retriggerJob = function (jobs) {
+        $scope.spinButton = function (selector, spinclass) {
+            $( selector ).removeClass( spinclass );
+            $( selector ).addClass( spinclass );
+            $timeout(function () {
```

Comment:
When I remove the timeout, I only get a single animation effect, no matter how many times I click the button.